apps/docs/src/app/fn/component-docs/utilities/focusable-list/focusable-list-header/focusable-list-header.component.html

```diff
@@ -0,0 +1,11 @@
+<header>Focusable List</header>
+<description>
+    You can quickly build out focusable list collection for your components. For that you will need to use combination
+    of <code>[fnFocusableList]</code> and <code>[fnFocusableItem]</code>. It has vertical and horizontal directions, as
+    well as respects <code>[fnDisabled]</code> and <code>[fnReadonly]</code> directives, so that those elements will be
+    excluded from the selection sequence.
```

Comment:
```suggestion
    You can quickly build out focusable list collection for your components. For that you will need to use a combination
    of <code>[fnFocusableList]</code> and <code>[fnFocusableItem]</code> directives. It has vertical and horizontal directions, and
    it also respects <code>[fnDisabled]</code> and <code>[fnReadonly]</code> directives, so that those elements will be
    excluded from the selection sequence.
```